ngraph/python/tests/test_ngraph/test_frontend_onnx.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2021 Intel Corporation
```

Comment:
Please create a new subdirectory for these tests in `ngraph/python/tests/test_frontend/`